tahrir/endpoints/__init__.py

```diff
@@ -0,0 +1,4 @@
+from flask import Blueprint
+
+
+blueprint = Blueprint("endpoints", __name__)
```

Comment:
These endpoints seem to be all about the API, so I suggest calling this blueprint "api" instead, I think it makes it clearer what it's about.